examples/uploadImageFile.js

```diff
@@ -0,0 +1,51 @@
+const umbraco = require("@umbraco/headless-client");
+const Client = umbraco.Client;
+const FormData = umbraco.FormData.default;
```

Comment:
The exported `FormData` will be removed in the next release since it's coming from the [form-data](https://www.npmjs.com/package/form-data) package and shouldn't really be re-exported by our library.

It should be possible to replace the it with the following

```js
const { Client } = require("@umbraco/headless-client");
const FormData = require("form-data");
```

Reply:
That's a good point. I assumed you exported for some specific compatibility reasons. :)